tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend_test_ci-pipeline.yml

```diff
@@ -0,0 +1,32 @@
+trigger: none
+
+jobs:
+- template: templates/linux-ci.yml
+  parameters:
+    AgentPool : 'ORT-Training-Frontend-Linux-GPU'
+    JobName: 'Onnxruntime_Linux_GPU_FrontEnd_Training'
+    SubmoduleCheckoutMode: 'recursive'
+    # TODO --skip_onnx_tests is used to disable onnx_test_runner test.
+    # onnx_test_runner test occasionally causes "CUDA failure 77: an illegal memory access was encountered"
+    # remove --skip_onnx_tests when the test failures are fixed.
+    BuildCommand: >
+      tools/ci_build/github/linux/run_dockerbuild.sh
+      -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory)
+      -x "
+      --enable_training
+      --config $(buildConfig)
+      --skip_onnx_tests
+      --build_wheel
+      --enable_frontend_training_tests
+      "
+    DoNugetPack: 'false'
+    ArtifactName: 'drop-linux'
+    TimeoutInMinutes: 90
+    Strategy: 
+      maxParallel: 2
+      matrix:
```

Comment:
is it necessary to run in both configurations? this queues two jobs. if we are concerned about build machine resources at the moment, maybe one is sufficient